apps/openmw/mwmechanics/obstacle.cpp

```diff
@@ -50,6 +50,15 @@ namespace MWMechanics
             const MWWorld::LiveCellRef<ESM::Door>& ref = *it;
 
             osg::Vec3f doorPos(ref.mData.getPosition().asVec3());
+
+            MWWorld::Ptr doorPtr = MWWorld::Ptr(&const_cast<MWWorld::LiveCellRef<ESM::Door> &>(ref), actor.getCell());
```

Comment:
Ugh! const_cast. Not yours, I see. But still. At least leave the FIXME comment in.